README.yaml

```diff
@@ -78,41 +78,33 @@ usage: |-
 
 examples: |-
   ```hcl
-  data "aws_iam_policy_document" "fluentd_user_policy" {
-    statement {
-      actions = [
-        "logs:DescribeDestinations",
-        "logs:DescribeExportTasks",
-        "logs:DescribeLogGroups",
-        "logs:DescribeLogStreams",
-        "logs:DescribeMetricFilters",
-        "logs:DescribeSubscriptionFilters",
-        "logs:FilterLogEvents",
-        "logs:GetLogEvents",
-        "logs:CreateLogGroup",
-        "logs:CreateLogStream",
-        "logs:PutLogEvents",
-        "logs:DescribeLogStreams",
-        "logs:CreateLogStream",
-        "logs:DeleteLogStream",
-      ]
-
-      resources = ["*"]
-    }
-  }
-
   module "fluentd_user" {
     source    = "git::https://github.com/cloudposse/terraform-aws-iam-system-user.git?ref=master"
     namespace = "eg"
     stage     = "dev"
     name      = "fluentd"
+
+    policy_arns_map = {
+      logs = "arn:aws:iam::aws:policy/CloudWatchLogsFullAccess"
+    }
+
+    inline_policies_map = {
+      s3 = aws_iam_policy_document.s3_policy.json
+    }
   }
 
+  data "aws_iam_policy_document" "s3_policy" {
+    statement {
+      actions = [
+        "s3:PutObject",
+        "s3:GetObjectAcl",
+        "s3:GetObject",
+        "s3:ListBucket",
+        "s3:PutObjectAcl"
+      ]
 
-  resource "aws_iam_user_policy" "default" {
-    name   = "${module.fluentd_user.user_name}"
-    user   = "${module.fluentd_user.user_name}"
-    policy = "${data.aws_iam_policy_document.fluentd_user_policy.json}"
+      resources = [ "arn:aws:s3:::bucket_name/*", "arn:aws:s3:::bucket_name/" ]
```

Comment:
I know it's more related to styling, but I think the multiline will be much pretty

Reply:
@SweetOps Done 👍